ocaml/xapi/export_raw_vdi.ml

```diff
@@ -51,14 +45,29 @@ let localhost_handler rpc session_id vdi (req: Http.Request.t) (s: Unix.file_des
              raise (Api_errors.Server_error (Api_errors.vdi_io_error, ["Device I/O errors"])) in
          begin
            try
-             Sm_fs_ops.with_block_attached_device __context rpc session_id vdi `RO
-               (fun path ->
-                  match Importexport.base_vdi_of_req ~__context req with
-                  | Some base_vdi ->
-                    Sm_fs_ops.with_block_attached_device __context rpc session_id base_vdi `RO
-                      (fun base_path -> copy (Some base_path) path)
-                  | None -> copy None path
-               )
+             let headers = Http.http_200_ok ~keep_alive:false () @ [
+                 Http.Hdr.task_id ^ ":" ^ (Ref.string_of task_id);
+                 Http.Hdr.content_type ^ ":" ^ content_type;
+                 Http.Hdr.content_disposition ^ ": attachment; filename=\"" ^ filename ^ "\""
+               ] in
+             Http_svr.headers s headers;
```

Comment:
You could make a `send_headers` function and call it from both `copy` where it used to be called previously, and in the `Tar` branch as well to keep the old behaviour.